windows-nsis/INSTALL-win32.txt

```diff
@@ -0,0 +1,62 @@
+GENERAL QUICKSTART FOR WINDOWS
+
+OpenVPN requires a configuration file and key/certificate files. You should 
+obtain these and save them to OpenVPN's configuration directory, usually 
+C:\Program Files\OpenVPN\config. Starting with OpenVPN 2.4-alpha1 you can also 
+put user-specific configurations to %USERPROFILE%\OpenVPN\config when using 
+OpenVPN GUI.
+
+You can run OpenVPN as a Windows system service, by using OpenVPN GUI or from a 
+command-prompt. To use the OpenVPN GUI, double click on the desktop icon or 
+start menu icon. OpenVPN GUI is a system-tray applet, so an icon for the GUI 
+will appear in the lower-right corner of the screen. Right click on the system 
+tray icon, and a menu should appear showing the names of your OpenVPN 
+configuration files, and giving you the option to connect. Starting in OpenVPN 
+2.4-alpha1 it is no longer necessary to have admin privileges to run the GUI, as 
+long as the OpenVPN Interactive Service is running. For details please refer to 
+OpenVPN GUI's documentation:
+
+    https://github.com/OpenVPN/openvpn-gui
+
+OpenVPN 2.4-alpha1 and later come bundled with three system services:
+
+    OpenVPNService (new, improved background service)
+    OpenVPNServiceInteractive (to run GUI as non-admin)
+    OpenVPNServiceLegacy (deprecated background service)
+
+The installer starts OpenVPNServiceInteractive automatically and configures it 
+to start at system startup. This is done to allow unprivileged users to start 
+OpenVPN connections using OpenVPN GUI without any extra configuration.
+
+The new OpenVPNService is a complete rewrite, and is able restart crashed 
+OpenVPN connections. It also works properly on new Windows versions unlike 
+OpenVPNServiceLegacy, which is included mostly for historic reasons.
+
```

Comment:
Well, I'm not sure that we even should include OpenVPNServiceLegacy, or that "historic" reasons is a valid reason. However, the decision was made to include it, so in it goes. The theory probably was that if openvpnserv2 somehow manages to behave worse than old openvpnserv's "automatic" part, then user can fall back to OpenVPNServiceLegacy. I'll modify the text to make that clear.